.github/workflows/publish.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           z=$(TZ=UTC0 git show ${{ github.sha }} --no-patch --format=%cd --date=format-local:'%s');
           ts=$(TZ=UTC0 git show ${{ github.sha }} --no-patch --format=%cd --date=format-local:'%Y%m%d%H%M');
           sha=$(echo "${{ github.sha }}" | cut -c1-7);
-          sed -i -r "s/^(version: .*)\.0$/\1.${z}+${ts}.git${sha}/" galaxy.yml;
+          sed -i -r "s/^(version: .*)\.EPOCH$/\1.${z}+${ts}.git${sha}/" galaxy.yml;
```

Comment:
I don't think this is needed because the patch version is in galaxy.yml

Reply:
you are right. what about modifying the galaxy.yml to have .EPOCH at the end to be consistent?

Reply:
It would prevent someone from building it locally and it will also get in the way of the CI https://github.com/redhat-cip/dci-openshift-agent/blob/master/test-runner#L466